### PR TITLE
Update cloud-recording.md

### PR DESCRIPTION
**Pull request by SamChung-MS**
Updated the documentation on how AMS Recordings are retained and affected by deletions of chat messages.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/cloud-recording.md [hunk 1]
```diff
@@ -50,7 +50,10 @@ For a Teams user's meetings to be recorded, Microsoft Stream must be enabled for
...
+
+> [!IMPORTANT]
+> Also note, for recordings that are on AMS, the recording retention is affected by the chat message itself. As such, any deletion of the original AMS Recording chat message  will prevent users from being able to access the recording. There are two scenarios that can affect this. 1) User manually deletes the chat messageare - In this scenario, as the original message is gone, users will no longer be able to access the recording and no further downloads will be possible. However, the recording itself may still be retained within Microsoft's internal systems for a time (not exceeding the original 21 day period). 2) Recording chat message is deleted by chat retention policy - AMS Recordings are direclty tied to the chat retention policy. As such, although recordings on AMS will by default be retained for 21 days before being deleted, if the chat message is deleted before the 21 days time period, due to chat message retention policies, the recording as well will be deleted. There is no way to recover the recording after this.
```
Comment (JohanFreelancer9) [inline Teams/cloud-recording.md:56]:
```suggestion
> Also note, for recordings that are on AMS, the recording retention is affected by the chat message itself. As such, any deletion of the original AMS Recording chat message will prevent users from being able to access the recording. There are two scenarios that can affect this. 1) User manually deletes the chat message - In this scenario, as the original message is gone, users will no longer be able to access the recording and no further downloads will be possible. However, the recording itself may still be retained within Microsoft's internal systems for a time (not exceeding the original 21-day period). 2) Recording chat message is deleted by chat retention policy - AMS Recordings are directly tied to the chat retention policy. As such, although recordings on AMS will by default be retained for 21 days before being deleted, if the chat message is deleted before the 21-day time period, due to chat message retention policies, the recording will also be deleted. There is no way to recover the recording after this.
```

Comment (JohanFreelancer9) [reply Teams/cloud-recording.md:56]: @SamChung-MS - please merge my suggested edits. Many thanks.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**